recipes/drwav/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class DrwavConan(ConanFile):
+    name = "drwav"
+    description = "WAV audio loader and writer."
+    homepage = "https://mackron.github.io/dr_wav"
+    topics = ("audio", "wav", "wave", "sound")
+    license = ("Unlicense", "MIT-0")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False],
+        "no_conversion_api": [True, False],
+        "no_stdio": [True, False]
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+        "no_conversion_api": False,
+        "no_stdio": False
+    }
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "dr_wav.c"]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def source(self):
+        info = self.conan_data["sources"][self.version]
+        tools.download(info["url"], "drwav.zip", sha256=info["sha256"])
+        for file in ("dr_wav.h", "LICENSE"):
+            tools.unzip("drwav.zip", destination=self._source_subfolder, pattern=os.path.join("*", file), strip_root=True)
+        os.unlink("drwav.zip")
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Reply:
Your suggestion would extract all files in the git repository which includes a lot of unrelated and unneeded files (the git repository hosts multiple unrelated libraries). The only two files needed are the header file and the LICENSE file which is why I went with the custom code to only extract those two files.

Reply:
keep same pattern as other recipes, besides, this is not a huge project like Boost, which takes hundreds of megabytes to install source